onchain/aiken/validators/active-operators.ak

```diff
@@ -85,6 +88,72 @@ validator spend(
           Key(operator) == parsed_active_node_output_datum.key,
         }
       }
+      UpdateBondHoldNewSettlement {
+        active_node_output_index,
+        hub_oracle_ref_input_index,
+        settlement_queue_input_index,
+        settlement_queue_redeemer_index,
+        new_bond_unlock_time,
+      } -> {
+        expect Finite(valid_to) = self.validity_range.upper_bound.bound_type
+        //
+        expect Some(active_node_input) =
+          list.find(
+            self.inputs,
+            fn(input) { input.output_reference == input_out_ref },
+          )
+        expect Some(active_node_input_datum) = datumOpt
+        //
+        expect Some(hub_oracle_ref_input) =
+          list.at(self.reference_inputs, hub_oracle_ref_input_index)
+        expect InlineDatum(hub_oracle_datum) = hub_oracle_ref_input.output.datum
+        expect parsed_hub_oracle_datum: HubOracleDatum = hub_oracle_datum
+        //
+        expect Some(active_node_output) =
+          list.at(self.outputs, active_node_output_index)
+        expect InlineDatum(active_node_output_datum) = active_node_output.datum
+        expect parsed_active_node_output_datum: NodeDatum =
+          active_node_output_datum
+        expect active_node_operator_output_data: Datum =
+          parsed_active_node_output_datum.data
+        //
+        expect Some(settlement_queue_redeemer_pair) =
+          list.at(self.redeemers, settlement_queue_redeemer_index)
+        expect AttachResolutionClaim { operator, .. } =
+          settlement_queue_redeemer_pair.2nd
+        // TODO fix performance:
+        expect Some(settlement_queue_input) =
+          list.at(self.inputs, settlement_queue_input_index)
+        expect
+          settlement_queue_input.output.address.payment_credential == parsed_hub_oracle_datum.settlement_queue_addr.payment_credential
+        expect
+          settlement_queue_redeemer_pair.1st == Spend(
+            settlement_queue_input.output_reference,
+          )
+        and {
+          no_active_operators_minted(
+            self.mint,
+            active_operators_mint_script_hash,
+          ),
+          ordered.prove_is_member(
+            active_operators_mint_script_hash,
+            parsed_active_node_output_datum.key,
+            active_node_input.output,
+          ),
+          //keys are already checked by prove_is_member()
+          parsed_active_node_output_datum.link == active_node_input_datum.link,
+          valid_to + maturity_duration == new_bond_unlock_time,
```

Comment:
Shouldn't we add `settlement_resolution_duration` here?
```suggestion
          valid_to + settlement_resolution_duration == new_bond_unlock_time,
```

Reply:
No, we've collapsed `maturity_duration` and `settlement_resolution_duration` into one parameter. There's no reason to separate them, and it makes our life easier in the operator directory.